changelog/17011.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+cli: Example vault read / write updated in help page.
```

Comment:
Can you please change to something like:
`cli: Example 'help' page KV read / write docs improved.`